staging/src/k8s.io/apimachinery/pkg/util/httpstream/wsstream/conn.go

```diff
@@ -272,6 +277,78 @@ func (conn *Conn) handle(ws *websocket.Conn) {
 	}
 }
 
+// handleV5 handles websocket data according to "V5" of the remotecommand subprotocol. The main
+// differences between V4 and V5 "handle" function is 1) a half-close signal on STDIN channel closes
```

Comment:
I think that the protocol ideally should be agnostic of the channels and implement per channel close or reset, not leaking the type of channel detail to the API

Reply:
Just to close the loop. After discussion offline, we have agreed that the V5 protocol supports a RESET signal, which is sent from the client and handled by the server by half-closing the STDIN stream. This allows the other streams to complete transmission, and the connection closes gracefully. The RESET signal is the max byte value (255) sent as the first byte of the data message (where the channel is). Please let me know if this adequately describes our agreement.